.github/workflows/windows.yaml

```diff
@@ -44,4 +43,4 @@ jobs:
           key: ${{ runner.os }}-protocompile-ci-${{ hashFiles('**/go.sum') }}
       - name: Test
         shell: bash
-        run: ./windows/test.bash
+        run: make protoc && go test ./...
```

Comment:
Unfortunately, that does not work. I put a blurb in the PR description about it. The `test` target depends on the `generate` target, but the `generate` target doesn't work on Windows. I already sunk a little time exploring, trying to get it to work, and decided it just doesn't matter. IMO, it's okay if `make generate` doesn't work on Windows, especially since it never has (so this PR is actually improving the status quo for Windows contributors, but isn't trying to fix everything).